src/view/views/ChallengeView.tsx

```diff
@@ -31,6 +55,8 @@ export const ChallengeView = () => {
         classes={["challenge-view--content"]}
       />
       <Divider />
+      <p>Status: {rst}</p>
+      <Divider />
```

Comment:
This should probably be an icon in green/red/grey based on the status or something.

Probably we also want to show this state in the challenge overview page.